job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -335,6 +337,42 @@ definitions:
         example:
           "tag:4": "4"
           owner: "bob"
+  expiration-policy:
+    type: object
+    properties:
+      "Active":
+        $ref: "#/definitions/policy"
+      "Completed":
+        $ref: "#/definitions/policy"
+      "Failed":
+        $ref: "#/definitions/policy"
+      "Cancelled":
+        $ref: "#/definitions/policy"
+      "Waiting":
+        $ref: "#/definitions/policy"
+      "Paused":
+        $ref: "#/definitions/policy"
+    description: The expiration policy to be applied on the job
+  policy:
+    type: object
+    properties:
+      referenceDate:
+        type: string
+        enum:
+          - createDate
+          - lastUpdateDate
+        default: createDate
+      duration:
+        type: integer
+        format: int32
+        default: 5
+      action:
+        type: string
+        enum:
+          - keep
+          - delete
+        default: keep
```

Comment:
Would `expire` be a better word than `keep`?

Reply:
Done